app/pages/[teamSlug]/team/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { useRouter } from 'next/router'
+import { Button } from '../../../frontend/components/button/button'
+import { TeamForm } from '../../../frontend/components/teamForm/teamForm'
+import { useTeamQuery, useUsersQuery } from '../../../frontend/generated/graphql'
+
+const Team = (): JSX.Element => {
+  const router = useRouter()
+  const [{ data: usersData, error: usersError }] = useUsersQuery()
```

Comment:
I think we only want to load the users that are members of the team and not all users.

What about adding the `members` field to the `Team` type in the `teamsBySlug` query?